hudi-spark/src/main/scala/org/apache/hudi/DataSourceOptions.scala

```diff
@@ -264,6 +264,7 @@ object DataSourceWriteOptions {
   val HIVE_PARTITION_EXTRACTOR_CLASS_OPT_KEY = "hoodie.datasource.hive_sync.partition_extractor_class"
   val HIVE_ASSUME_DATE_PARTITION_OPT_KEY = "hoodie.datasource.hive_sync.assume_date_partitioning"
   val HIVE_USE_PRE_APACHE_INPUT_FORMAT_OPT_KEY = "hoodie.datasource.hive_sync.use_pre_apache_input_format"
+  val HIVE_JDBC_ENABLE_OPT_KEY = "hoodie.datasource.hive_sync.usejdbc"
```

Comment:
We would need to document this in configurations page in website.
@n3nash : Can you add a follow up PR (Jira: https://jira.apache.org/jira/browse/HUDI-883) when you get a chance.

Reply:
sounds good, will do